src/stores/users/userActionCreators.js

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios';
+import * as ActionTypes from '../actionTypes';
+
+export const signIn = (user) => ({
+  type: ActionTypes.USER_SIGNIN,
+  payload: user,
+});
+
+export const userLogout = (user) => ({
+  type: ActionTypes.USER_LOGOUT,
+  payload: user,
+});
+
+export const postSignIn = (email, password) => (dispatch) => {
+  axios
+    .post('http://localhost:3000/v1/users/login', {
+      email,
+      password,
+    })
+    .then((response) => {
+      localStorage.setItem('auth', JSON.stringify(response.data));
```

Comment:
If you need to save the store because the user refreshes the page, there is a middleware for redux similar to thunk to store the whole Redux state. This way you don't have to implement this logic of storing things in the localStorage